packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -76,15 +85,47 @@ export interface EditableTree {
     readonly [proxyTargetSymbol]: object;
 
     /**
-     * Anchor to this node.
-     * Valid as long as this EditableTree's context is not freed.
-     * Might not point to any node if this node is deleted from the document.
-     *
-     * TODO: When a proper editing API is exposed on EditableTree directly,
-     * this should become an implementation detail and rbe removed from this API surface.
+     * Creates a child of this node.
+     */
+    readonly [insertNodeSymbol]: (key: string, value: ITreeCursor) => boolean;
+
+    /**
+     * Sets value of a child of this node.
+     */
+    readonly [setValueSymbol]: (key: string, value: unknown, typeName: TreeSchemaIdentifier) => boolean;
+
+    /**
+     * Deletes a child of this node.
      */
-    readonly [anchorSymbol]: Anchor;
+    readonly [deleteNodeSymbol]: (key: string) => boolean;
```

Comment:
THis has most of the issues setValue has:

- Undocumented return value.
- uses string field key
- node talking about children on a type who's name implies no children
- cannot be used to delete a node without access to the node's parent (this might be ok if there is another APi for deleting in other cases)